VERSION

```diff
@@ -0,0 +1 @@
+1.4.1-SNAPSHOT
```

Comment:
Unfortunately, I don't think there is a way for package.json to read this file. So the for daffodil-vscode, version bumps require modifying two files, package.json and VERSION

Reply:
The action was intended to be run immediately after the checkout action and before yarn/npm/sbt/etc commands are run that could generate that file. I *guess* it might work to run something to generate the VERSION file in between checkout and asf-release-candidate actions, but it feels safer to run from a clean slate.

I'm open to other suggestions for the action to figure out the version, but I would like to avoid parsing build configs. That worked in the past because we only have three projects, but I really want this action to be general and simple enough that it could be used by other ASF projects so that maybe someday the maintenance could be shared.